src/http2_frame_headers.erl

```diff
@@ -91,3 +83,74 @@ from_frames([], Acc) ->
     Acc;
 from_frames([{#frame_header{type=?CONTINUATION},#continuation{block_fragment=BF}}|Continuations], Acc) ->
     from_frames(Continuations, <<Acc/binary,BF/binary>>).
+
+-spec split(Binary::binary(),
+            MaxFrameSize::pos_integer()) ->
+                   [binary()].
+split(Binary, MaxFrameSize) ->
+    split(Binary, MaxFrameSize, []).
+
+-spec split(Binary::binary(),
+            MaxFrameSize::pos_integer(),
+            [binary()]) ->
+                   [binary()].
+split(Binary, MaxFrameSize, Acc)
+  when byte_size(Binary) =< MaxFrameSize ->
+    lists:reverse([Binary|Acc]);
+split(Binary, MaxFrameSize, Acc) ->
+    <<NextFrame:MaxFrameSize/binary,Remaining/binary>> = Binary,
+    split(Remaining, MaxFrameSize, [NextFrame|Acc]).
+
+%% Now build frames.
+%% The first will be a HEADERS frame, followed by CONTINUATION
+%% If EndStream, that flag needs to be set on the first frame
+%% ?FLAG_END_HEADERS needs to be set on the last.
```

Comment:
I wrote so much code because I believed that too. It was great when I deleted it a few months ago.

Look at the spec: https://tools.ietf.org/html/rfc7540#section-6.10

CONTINUATION frames can't have the END_STREAM flag